client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -42,31 +119,41 @@ export default {
 
   components: {
     FieldPicker,
+    VueSelect,
   },
 
   extends: base,
+
+  computed: {
+    documentationURL () {
+      // eslint-disable-next-line no-undef
+      const [year, month] = VERSION.split('.')
+      return `https://docs.cortezaproject.org/corteza-docs/${year}.${month}/integrator-guide/compose-configuration/index.html`
+    },
+
+    fieldOptions () {
```

Comment:
vue-select can be configured so you dont need the map here, you could also avoid the filter by extending the selectable prop